evap/evaluation/auth.py

```diff
@@ -136,6 +136,15 @@ def check_user(user):
     return user_passes_test(check_user)(view_func)
```

Comment:
is `staff_required` used anywhere after applying this PR?

Reply:
This is not used yet, so I removed it.